app/views/service_providers/_form.html.erb

```diff
@@ -133,12 +131,12 @@
       <%= form.label :allow_prompt_login,
                       label: 'Enable <prompt=login>',
                       class: 'usa-label' %>
-      <ul class="usa-input-list">
-        <li>
+      <div>
```

Comment:
Same question about `<div>` wrapper. There's another example below too.

Reply:
updated in https://github.com/18F/identity-dashboard/pull/740/commits/20e94541458f876cf2ab980a6a9c511e0bce11dc